src/prefect/cli/cloud/__init__.py

```diff
@@ -515,6 +515,30 @@ async def logout():
     exit_with_success("Logged out from Prefect Cloud.")
 
 
+@cloud_app.command()
+async def open():
+    """
+    Open the Prefect Cloud UI in the browser.
+    """
+    confirm_logged_in()
+
+    current_profile = prefect.context.get_settings_context().profile
+    if current_profile is None:
+        exit_with_error("There is no current profile set.")
```

Comment:
What do you think of adding some prescriptive text here to let the user know what they should do to get this working?